app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
```

Comment:
Invalid indentation. Presence of `-d` argument  doesn't depend on presence of `-f` argument
```suggestion
if "-d" in argv:
    index2 = 0

    for _ in range(len(argv) - 1):
        index2 += 1
        if argv[_: _ + 1] == ["-d"]:
            break
```